iceoryx_posh/test/moduletests/test_popo_chunk_queue.cpp

```diff
@@ -144,56 +146,52 @@ TEST_P(ChunkQueue_test, ClearWithData)
     EXPECT_THAT(m_popper.empty(), Eq(true));
 }
 
-TEST_P(ChunkQueue_test, AttachSemaphore)
+TEST_P(ChunkQueue_test, AttachConditionVariableSignaler)
 {
-    auto semaphore = semaphorePool.createObjectWithCreationPattern<iox::posix::Semaphore::errorType_t>(0);
-    ASSERT_THAT(semaphore.has_error(), Eq(false));
+    ConditionVariableData condVar;
 
-    auto ret = m_popper.attachSemaphore(*semaphore);
-    EXPECT_FALSE(ret.has_error());
+    auto ret = m_popper.attachConditionVariableSignaler(&condVar);
+    EXPECT_TRUE(ret);
 
-    EXPECT_THAT(m_popper.isSemaphoreAttached(), Eq(true));
+    EXPECT_THAT(m_popper.isConditionVariableSignalerAttached(), Eq(true));
 }
 
 TEST_P(ChunkQueue_test, DISABLED_PushAndTriggersSemaphore)
 {
-    auto semaphore = semaphorePool.createObjectWithCreationPattern<iox::posix::Semaphore::errorType_t>(0);
-    ASSERT_THAT(semaphore.has_error(), Eq(false));
-
-    auto ret = m_popper.attachSemaphore(*semaphore);
-    EXPECT_FALSE(ret.has_error());
+    ConditionVariableData condVar;
+    ConditionVariableWaiter condVarWaiter{&condVar};
 
-    EXPECT_THAT(semaphore->get()->tryWait(), Eq(false));
+    auto ret = m_popper.attachConditionVariableSignaler(&condVar);
+    EXPECT_TRUE(ret);
 
     auto chunk = allocateChunk();
     m_pusher.push(chunk);
 
-    EXPECT_THAT(semaphore->get()->tryWait(), Eq(true));
-    EXPECT_THAT(semaphore->get()->tryWait(), Eq(false)); // shouldn't trigger a second time
+    EXPECT_THAT(condVarWaiter.timedWait(1_ms), Eq(true));
+    EXPECT_THAT(condVarWaiter.timedWait(1_ms), Eq(false)); // shouldn't trigger a second time
 }
 
 TEST_P(ChunkQueue_test, DISABLED_AttachSecondSemaphore)
```

Comment:
Rename test for condition variable. Is this test still required as it's disabled?

Reply:
Renamed the test case. See answer above.